Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.0.1'
```

Comment:
I don't quite follow. As I understand it, the Gemfile.lock should keep all our different deployments in sync: http://stackoverflow.com/questions/6927442/what-is-the-difference-between-gemfile-and-gemfile-lock-in-ruby-on-rails. Am I missing something?